### ISSUE TITLE
[VS test stability] LAG TPID 

### ISSUE THREAD

**Issue by prsunny** [Azure/sonic-swss#1779]
LAG TPID test is unstable, please fix stability

`test_portchannel.py::TestPortchannel::test_Portchannel_tpid FAILED  `

```
2021-06-10T18:41:49.9930068Z         # Check ASIC DB
2021-06-10T18:41:49.9930546Z         # get TPID and validate it to be 0x9200 (37376)
2021-06-10T18:41:49.9934136Z         atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_LAG")
2021-06-10T18:41:49.9935071Z         lag = atbl.getKeys()[0]
2021-06-10T18:41:49.9935511Z         (status, fvs) = atbl.get(lag)
2021-06-10T18:41:49.9935991Z         assert status == True
2021-06-10T18:41:49.9936557Z         asic_tpid = "0"
2021-06-10T18:41:49.9936907Z     
2021-06-10T18:41:49.9937295Z         for fv in fvs:
2021-06-10T18:41:49.9937744Z             if fv[0] == "SAI_LAG_ATTR_TPID":
2021-06-10T18:41:49.9938527Z                 asic_tpid = fv[1]
2021-06-10T18:41:49.9938872Z     
2021-06-10T18:41:49.9939226Z >       assert asic_tpid == "37376"
2021-06-10T18:41:49.9940348Z E       AssertionError: assert '0' == '37376'
2021-06-10T18:41:49.9941103Z E         - 0
2021-06-10T18:41:49.9941520Z E         + 37376
```

**Comment by wendani** [comment]
Should be fixed by https://github.com/Azure/sonic-swss/pull/1767

**Comment by prsunny** [comment]
> Should be fixed by #1767

I see, thanks @wendani
